server/routes/prisonerReport/prisonerReportReview.ts

```diff
@@ -35,7 +35,7 @@ export default class prisonerReportReviewRoutes {
       prisoner,
       data,
       printHref: `/print-report/${adjudicationNumber}?referrer=/prisoner-report/${prisoner.prisonerNumber}/${adjudicationNumber}/review`,
-      editIncidentDetailsURL: `/incident-details/${prisoner.prisonerNumber}/${data.draftId}/submitted/edit`,
+      editIncidentDetailsURL: `/incident-details/${prisoner.prisonerNumber}/${data.draftId}/submitted/edit?referrer=/prisoner-report/${prisoner.prisonerNumber}/${adjudicationNumber}/review`,
```

Comment:
Maybe it's not important, but I would have expected tests to fail - and be changed - because of this?!